src/common/common-types.ts

```diff
@@ -409,3 +445,12 @@ export interface RgbaColorJson {
     readonly kind: 'rgba';
     readonly values: readonly [r: number, g: number, b: number, a: number];
 }
+
+export interface CacheSettingValue {
+    enabled: boolean;
+    location: string;
+}
+
+export interface PackageSettings {
+    [packageName: string]: Record<string, string | number | boolean | CacheSettingValue>;
+}
```

Comment:
Man, I don't like cache setting. It's so specialized... Not sure what to do about it though.

Reply:
I also don't like it. The alternative was to have a standard path convention that the frontend and backend would both find independently, but I didn't like the potential desync. Idk what a better solution would be